views/drones/list.hbs

```diff
@@ -1 +1,28 @@
-<h2>Available drones</h2>
\ No newline at end of file
+
+<div class="Container-fluid d-flex flex-column ">
+    <h2>Available drones</h2>
+    <button class="btn-list-1"><a href="/">Home</a></button>
+        <div class="row"></div>
+        {{#each drones}}
+            <div class="p-2 d-flex justify-content-center drones">
+                <h2> Name: {{this.name}}</h1>
+                <div class="p-2 d-flex justify-content-center">
+                    <p class="p-tag"> | Propellers Number: {{this.propellers}}</p>
+                    <p class="p-tag"> | Maximum Speed: {{this.maxSpeed}}</p>
+                </div>
+            </div>
+            <div class="p-2 d-flex justify-content-center">
+                <button class="btn-list"><a href="/drones/{{this.id}}/edit">Modify your Drone</a></button>
```

Comment:
I think It would be better to encapsulate the `<button>` with your `<a>` as it makes it easier for the user (with what you have now the user would have to click on "Modify Your Drone" but he'll stay on the page if he clicks on the button itself which isn't the best in terms of UI interaction).
```
<a href="/drones/{{this.id}}/edit"><button class="btn-list">Modify your Drone</button></a>